lib/galaxy/dependencies/pinned-requirements.txt

```diff
@@ -175,6 +175,7 @@ stevedore==3.3.0; python_version >= "3.6"
 svgwrite==1.4.1; python_version >= "3.6"
 tempita==0.5.2
 tenacity==6.3.1
+tifffile==2021.2.26
```

Comment:
Can you also add the unpinned dependency to https://github.com/galaxyproject/galaxy/blob/dev/pyproject.toml ?

Reply:
Done